forms-flow-theme/scss/_modal.scss

```diff
@@ -557,3 +557,33 @@
  gap: var(--spacer-150);
  align-self: stretch;
 }
+.build-modal{
+  width: 60.318% !important;
+  max-width: 60.318% !important;
```

Comment:
may be we have to update this on later stage. will merge for now @Ajay-aot

Reply:
okay